mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -41,8 +41,10 @@ def __init__(self, tunables: TunableGroups, service: Optional[Service], config:
         self._tunables = tunables
         self._service = service
         self._iter = 1
-        self._use_defaults: bool = bool(strtobool(str(self._config.pop('use_defaults', True))))
-        self._max_iter = int(self._config.pop('max_iterations', 25))
+        # If True, suggest the already assigned tunable values for the first iteration:
```

Comment:
```suggestion
        # If True, suggest the "already assigned tunable values" for the first iteration:
```

"already assigned tunable values" means something slightly different than "start_with_defaults".

We should probably document that somewhere for the user.  For instance:
> "start_with_defaults" means initialize the first iteration with the `tunable_params` defaults, and possibly overlay those with any `tunable_values` that were provided.

Reply:
done